src/main/java/uk/gov/companieshouse/model/validator/AppealReasonValidator.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.model.validator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.exception.AppealReasonException;
+import uk.gov.companieshouse.model.Reason;
+
+@Component
+public class AppealReasonValidator {
```

Comment:
I'm not particularly infused by having a secondary validation system that isn't tied into the existing validation. Why not integrate this into the Spring validation?

Reply:
Its also deceptive because it is totally unrelated to any validation annotations on the class which can add further confusion for anyone who has to pick up this code.

Reply:
Thanks Jonathan! and I agree, but, 
for maintainability and readability it will make sense to have a validation on a method or field level,
but it is hiding most of the logic and I would like to avoid it. Simplicity it is what I like most especially 
for someone new in Java :).
Anyway I do not want to add complexity now when we have to move most of these logic down
to CHIP interface

Reply:
So Mo, in that case do you think other model classes should be moved onto custom validators like this and Spring annotations should be phased out?

Reply:
Nope, but lets keep things simple

Reply:
I agree with the spirit, I disagree with what you consider to be more simple.